gnssanalysis/gn_io/igslog.py

```diff
@@ -109,21 +134,25 @@ def parse_igs_log(filename_array: _np.ndarray) -> _np.ndarray:
     with open(file_path, "rb") as file:
         data = file.read()
 
-    blk_id = _REGEX_ID.search(data)
+    blk_id = _REGEX_ID_V2.search(data)
     if blk_id is None:
```

Comment:
Maybe a variable `version_2` can be used to have so we don't need to do the check for the next modification

Reply:
Left this as is, introduced an if/else above to assign `_REGEX_ID` ...